### PR TITLE
Added documentation around Enhanced Dev Preview of ACM Right Sizing

### PR DESCRIPTION
**Pull request by rajusem**
Added below pages for the Enhanced dev preview of ACM Right Sizing

- Overview of Architecture 
- Installation steps to enable Right Sizing component 
- How to use grafana dashboard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,11 @@ Output :
...
 
 For more details to understand on how the system works, proceed [here](doc/details.md)   
+
+# Enhanced Dev Preview
+
+The output of the thanos-metrics-analyzer is a **CSV** file with max usage values and recommendations for CPU and memory utilization. **No charts** were made available to customers in the RHACM console.
+
+With the new enhancements made to the developer preview experience, including simplified user navigation directly in the RHACM console, **Grafana dashboards** are now enriched with right-sizing recommendations for cluster as well as namespace level based on CPU and memory information. 
+
+For more details checkout [this](doc/enhance-dev-preview-overview.md) page.
```
Comment (Anxhela21) [inline README.md:31]: Should we move this to the top of the file?

File: doc/enhanced-dev-preview-installation-steps.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Prerequisite 
+- Openshift Cluster (Tested with 4.15.13+ version)
+- Advance Cluster Management operator should be installed for ACM hub/spoke feature
```
Comment (Anxhela21) [inline doc/enhanced-dev-preview-installation-steps.md:3]:
```suggestion
- The Advanced Cluster Management operator should be installed on the hub cluster to facilitate cluster administration and data aggregation.
```

File: README.md (hunk 1 above)
Comment (rajusem) [reply README.md:31]: Purposefully I kept below, as it has `thanos-metrics-analyzer` reference, so it should go first.

Comment (Anxhela21) [reply README.md:31]: ah okay having two different names - makes sense to go second

**Review by Anxhela21 — APPROVED** [review]
Looks good. Left some minor comments. Please unhold when ready to merge.

/lgtm
/hold

**Pull request merged**